app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -24,6 +24,16 @@ def x_view_toolbar_filename
     end
   end
 
+  NO_SUMMARY_VIEW_TB = [
```

Comment:
This is some additional complexity that can cause us a lot of headache in the future. What are these below? I don't see only chargeback-related stuff, so I can't really suggest you how else to do this.

Reply:
@skateman this part of code renders download summary screen button for most of the summary screens. Only handful of screens don't need that button like Request summary, chargeback rates summary, timelines(need to test that) and there are some redundant ones in the list that can be removed. Should i change this code to `elsif !@no_summary_view_tb && @display == "main" && @showtype == "main" && !@in_a_form` and then set `@no_summary_view_tb = true` in those controllers, I don't know how else I can implement this without making changes to other controllers that actually need to show this button.

Reply:
Oh, so this is just something extracted. I don't like the added complexity, but it's not added by you, so I guess it's okay.